IBPSA/Fluid/HeatExchangers/BaseClasses/PartialEffectivenessNTU.mo

```diff
@@ -105,6 +105,11 @@ initial equation
     Q_flow_nominal = -m2_flow_nominal*cp2_nominal*(T_a2_nominal - T_b2_nominal);
     eps_nominal = abs(Q_flow_nominal/((T_a1_nominal - T_a2_nominal)*
       CMin_flow_nominal));
+    assert(Q_flow_nominal / (T_a1_nominal - T_a2_nominal) >= 0,
+    "In " + getInstanceName() + ": Q_flow_nominal is defined with the wrong sign. " +
+    "By convention, a positive value describes a heat flow from port 1 to port 2. " +
+    "Future version of this library might enforce this convention and throw an error.",
+    level = AssertionLevel.warning)
```

Comment:
@mwetter Are warnings accepted by the CI testing on MBL or IBPSA?
Otherwise use `Modelica.Utilities.Streams.print("Warning: ...")`

Reply:
@AntoineGautier : I suggest to use `level = AssertionLevel.warning`, and then make sure they don't trigger in the examples of the library.

Reply:
I ran some tests for IBPSA and Buildings (namely the tests in `Fluids.HeatExchangers` and in `Buildings.Examples`) and did not see the warnings pop up.

Reply:
Nervermind. The warnings didn't show up when running the tests from BuildingPy, but running them individually from Dymola, I get some warnings.

Reply:
Alright, I manually tracked wherever that class was being used (hopefully this was thorough), and I did not find any examples that raised the warning in the IBPSA repo. I did find a few in the MBL, which I am addressing in this PR:
https://github.com/lbl-srg/modelica-buildings/pull/2370

Otherwise, I think that this is ready to merge.